VoxelSniperCore/src/main/resources/lang.yml

```diff
@@ -4,8 +4,8 @@ TREESNIPE_USAGE: |-
   </gold><aqua>/b %triggerHandle% [treeType]  -- Change tree type
 SPLATTER_VOXEL_DISC_BRUSH_USAGE: |-
   <gold>Splatter Voxel Disc Brush Parameters:
-  </gold><aqua>/b %triggerHandle% seed [decimal]  -- Set a seed percentage
-  </aqua><aqua>/b %triggerHandle% growth [decimal]  -- Set a growth percentage
+  </gold><aqua>/b %triggerHandle% seed [number]  -- Set a seed percentage
+  </aqua><aqua>/b %triggerHandle% growth [number]  -- Set a growth percentage
```

Comment:
See comment from yesterday, in the current state, this won't affect existing installs.